src/Common/RJSF/widgets/Select.tsx

```diff
@@ -35,7 +35,7 @@ const DropdownIndicator = (props) => (
     </components.DropdownIndicator>
 )
 
-export const Select = (props: WidgetProps) => {
+export const SelectWidget = (props: WidgetProps) => {
```

Comment:
Are we not changing the select here?

Reply:
Will be picking that up in next PR where will be handling some variant which havent be done yet